DOCKER.md

```diff
@@ -0,0 +1,66 @@
+### Docker
+
+It is possible to setup the OpenFoodnetwork app easily with Docker and Docker Compose.
+The objective is to spare configuration time, in order to help people testing the app and contribute to it.
+It can also be use as documentation. It is not perfect but it is used in many other projects and many devs are used to it nowadays.
+
+### Install Docker
+
+Please check the documentation here, https://docs.docker.com/install/ to install Docker.
+
+For Docker Compose, information are here: https://docs.docker.com/compose/install/.
+
+Better to have at least 2GB free on your laptop in order to download images and create containers for OpenFoodNetwork app.
+
+
+### Use Docker with OpenFoodNetwork
+
+Open a terminal with a shell.
+
+Clone the repository:
+
+    git clone git@github.com:openfoodfoundation/openfoodnetwork.git
```

Comment:
I'd rather use backticks (```) to make use of markdown's code blocks and precede commands with `$` to make it more explicit. They are conventions I always follow in the wiki.

Reply:
Ok, I will check this out, for now I just followed how `GETTING_STARTED.md` is written.

Reply:
Good to know. Might be worth updating it